bin/github-backup

```diff
@@ -537,6 +537,24 @@ def _request_url_error(template, retry_timeout):
     return False
 
 
+def download_file(url, path, auth):
+    request = Request(url)
+    request.add_header('Accept', 'application/octet-stream')
+    request.add_header('Authorization', 'Basic '.encode('ascii') + auth)
+    data = urlopen(request)
```

Comment:
This blows up as a 400 bad request. I dug into `urllib` code a bit and found a little more information

```
<urllib.request.Request object at 0x7ff47036b048> <http.client.HTTPResponse object at 0x7ff470735940> 400 Bad Request x-amz-request-id: DA588EA498B00726
x-amz-id-2: XMwcw3BJHcGcrQk8QPkViVr+PtNAFs6rh881+mBtapSZsjiJhcJzgkWuEyS2LmeS/1Rgp4H+svY=
Content-Type: application/xml
Transfer-Encoding: chunked
Date: Sat, 22 Jun 2019 01:01:30 GMT
Connection: close
Server: AmazonS3
```

it seems like `urllib` is not handling the redirect to AWS S3 correctly, but I'm not sure.